disnake/client.py

```diff
@@ -1258,6 +1258,11 @@ def run(self, *args: Any, **kwargs: Any) -> None:
         function should not be used. Use :meth:`start` coroutine
         or :meth:`connect` + :meth:`login`.
 
+        Parameters
+        ----------
+        token: `str`
+            The discord token of the bot that is being ran.
```

Comment:
This block needs to be moved to the end of the docstring, since the "Roughly Equivalent To" part and the warning are meant to be part of the description (sphinx also [complains](https://github.com/DisnakeDev/disnake/actions/runs/11632240481/job/32394941932?pr=1240#step:4:360))